src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -104,21 +113,24 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
         Promise.allSettled([
             getRecentDeploymentConfig(appId, pipelineId),
             getLatestDeploymentConfig(appId, pipelineId),
-            getSpecificDeploymentConfig(appId, pipelineId, this.getWfrId()),
+            this.state.isRollbackTrigger ? getSpecificDeploymentConfig(appId, pipelineId, this.getWfrId()) : () => {},
```

Comment:
we can use noop instead of () => {}. as it is created for the same

Reply:
done